tests/unit/components/Camera/StandardCamera.test.js

```diff
@@ -17,6 +17,17 @@ jest.mock( "@react-navigation/native", () => {
   };
 } );
 
+jest.mock( "react-native-orientation-locker", () => ( {
+  addEventListener: jest.fn(),
+  addDeviceOrientationListener: jest.fn(),
+  removeEventListener: jest.fn(),
+  lockToPortrait: jest.fn(),
+  lockToLandscapeLeft: jest.fn(),
+  lockToLandscapeRight: jest.fn(),
+  unlockAllOrientations: jest.fn(),
+  removeOrientationListener: jest.fn()
+} ) );
+
```

Comment:
I can't think of a reason that we'd be able to ever use a non-mocked version of this library, so I think this belongs in `jest.setup.js`, or another global test setup area.